src/common/middleware/crashlytics.js

```diff
@@ -0,0 +1,53 @@
+import { setCustomKey } from "../crashlytics";
+
+// Helper function to safely stringify values
+const safeStringify = (value) => {
+  if (value === null || value === undefined) {
```

Comment:
Thinking out loud, If we write:
This should cover both null and undefined
```javascript
  if (value == null) return ""; 
```

Reply:
fixed